core/src/main/java/io/questdb/cutlass/text/CsvFileIndexer.java

```diff
@@ -633,12 +633,7 @@ void openInputFile() {
         }
 
         path.of(inputRoot).slash().concat(inputFileName).$();
-
-        long fd = ff.openRO(path);
-        if (fd < 0)
-            throw CairoException.instance(ff.errno()).put("could not open read-only [file=").put(path).put(']');
-
-        this.fd = fd;
+        this.fd = TableUtils.openRO(ff, path, LOG);
```

Comment:
Why is this fn in the TableUtils namespace? Shouldn't it be moved to a more suitable location?